lib/shopify_api/rest/collect.ex

```diff
@@ -4,18 +4,18 @@ defmodule ShopifyAPI.REST.Collect do
   """
 
   alias ShopifyAPI.AuthToken
-  alias ShopifyAPI.REST.Request
+  alias ShopifyAPI.REST
 
   @doc """
   Add a product to custom collection.
 
   ## Example
 
-      iex> ShopifyAPI.REST.Request.add(auth)
+      iex> ShopifyAPI.REST.REST.add(auth)
```

Comment:
This looks like it was wrong before, should actually be
`ShopifyAPI.REST.Collect.add(auth)`